.gitlab-ci.yml

```diff
@@ -1,11 +1,9 @@
 stages:
   - build
 
-# Used for ci setup in the gitlab mirror of the project:
-# https://gitlab.haskell.org/haskell/haskell-language-server/-/pipelines
 variables:
   # Commit of ghc/ci-images repository from which to pull Docker images
-  DOCKER_REV: "9e4c540d9e4972a36291dfdf81f079f37d748890"
+  DOCKER_REV: "572353e0644044fe3a5465bba4342a9a0b0eb60e"
 
   GHC_VERSION: 9.2.3
   CABAL_INSTALL_VERSION: 3.6.2.0
```

Comment:
BTW, this is untouched. Is this what GHC CI uses, or just random old versions? Is there any reason I can't bump it to 

```
  GHC_VERSION: 9.2.7
  CABAL_INSTALL_VERSION: 3.6.3.0
```

once the dust settles after the release?

Reply:
It's just random old stuff. :) Furthermore I think it should be detached from GHC anyway.

I just wouldn't make it *too* new --- you want to be able to bootstrap from older versions. In fact, having it be as old as possible is a good compatibility check, though of course you may want use newer versions to get whatever bug fixes, performance improvements, or shiny new  features you want.